src/mixed-reality/azext_mixed_reality/tests/latest/test_spatial_anchors_account.py

```diff
@@ -0,0 +1,69 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer)
+from knack.util import CLIError
+
+
+class SpatialAnchorsAccountScenarioTest(ScenarioTest):
+
+    @ResourceGroupPreparer(location='eastus2', parameter_name_for_location='location')
+    def test_spatial_anchors_account_scenario(self, resource_group, location):
+
+        name = self.create_random_name(prefix='cli', length=24)
+
+        self.kwargs.update({
+            'initial': 'az spatial-anchors-account',
+            'name': name,
+            'location': location,
+        })
+
+        try:
+            # Create
+            self.__assert_spatial_anchors_account_not_exist()
+
+            self.__assert_spatial_anchors_account_as_expected('{initial} create -g {rg} -n {name} -l {location}')
+
+            # Read
+            self.__assert_spatial_anchors_account_as_expected('{initial} show -g {rg} -n {name}')
+
+            # Primary Key
+            self.__assert_spatial_anchors_account_keys_work('primary', 'secondary')
+
+            # Secondary Key
+            self.__assert_spatial_anchors_account_keys_work('secondary', 'primary')
+
+            # Delete
+            self.__delete_spatial_anchors_account()
+            self.__assert_spatial_anchors_account_not_exist()
+
+        finally:
+            # Delete is idempotent
+            self.__delete_spatial_anchors_account()
+
+    def __assert_spatial_anchors_account_not_exist(self):
```

Comment:
If this method is checking that a specific spatial anchors account does not exist then the method should take the name of the specific spatial anchors account as a name. For example:

```suggestion
    def __assert_spatial_anchors_account_not_exist(self, sa_name):
```

Reply:
The helper method is to check the test Spatial Anchors Account of which name is randomly generated as {name}.